app/views/catalog/_search_targets_widget.html.erb

```diff
@@ -12,11 +12,20 @@
     </h1>
   </a>
    <ul class="dropdown-menu" role="menu">
-     <li class="<%= 'active' unless article_search? %>">
-       <%= link_to_catalog_search %>
-     </li>
-     <li class="<%= 'active' if article_search? %>">
-       <%= link_to_article_search %>
-    </li>
+     <% if article_search? %>
+       <li>
+         <%= link_to_catalog_search %>
+       </li>
+      <li class="active">
+        <%= I18n.t('searchworks.search_dropdown.articles.description_html').html_safe %>
+      </li>
+     <% else %>
+       <li class="active">
+         <%= I18n.t('searchworks.search_dropdown.catalog.description_html').html_safe %>
+       </li>
+       <li>
+         <%= link_to_article_search %>
+       </li>
+     <% end %>
```

Comment:
Hmm, I worry that having the references to the i18n keys both here and in the helper.

I wonder if this might be a good use for something like `link_to_if` or `link_to_unless` in the helper?